scripts/upload-apk.sh

```diff
@@ -21,17 +23,14 @@ else
 	/bin/rm -f badge-magic-dev-*.apk
 fi
 
-\cp -r ../app/build/outputs/apk/*/**.apk .
-
-# Signing Apps
-cp app-release-unsigned.apk app-release-unaligned.apk
-jarsigner -verbose -tsa http://timestamp.comodoca.com/rfc3161 -sigalg SHA1withRSA -digestalg SHA1 -keystore ../scripts/key.jks -storepass $STORE_PASS -keypass $KEY_PASS app-release-unaligned.apk $ALIAS
-${ANDROID_HOME}/build-tools/28.0.3/zipalign -v -p 4 app-release-unaligned.apk app-release.apk
+find ../app/build/outputs -type f \( -name '*.apk' -o -name '*.aab' \) -exec cp -v {} . \;
 
 #removing unused apps
 for file in app*; do
     if [[ ${file} =~ "unsigned" || ${file} =~ "unaligned" ]]; then
         rm ${file}
+    elif [[ ${file} =~ ".aab" ]]; then
```

Comment:
Master and dev aab should be separate

Reply:
Areeb, aab is a publishing format it is of no use to the normal users. Why do you want to keep in the apk branch?

Reply:
that is the reason I just used it to push the app to playstore and keep the release and debug apk's in the apk branch.

Reply:
It is of use to me. Because I want to debug what was sent to play store if something goes wrong.

Normal users use fdroid or play store to download the APK. I and fellow developers and maintainers use the APK branch

Reply:
nvm, ill do it right away